wordpress-mobile/AztecEditor-Android
# Move CI to Buildkite

**To Test:**
- Ensure all Buildkite CI checks pass.
- It's ok if there are CircleCI build errors – its configuration has been removed. Once this PR is approved I'll disconnect CircleCI entirely so the error doesn't propagate to the `trunk` branch.
